eslint-config/.eslintrc.es6.json

```diff
@@ -41,13 +41,15 @@
       {
         "either": "either" // control has to be either nested or associated via htmlFor
       }
-    ]
+    ],
+    "import/no-default-export": 1 // no default exports
   },
   "overrides": [
     {
       "files": ["*.spec.js"],
       "rules": {
-        "no-unused-expressions": "off"
+        "no-unused-expressions": "off",
+        "import/no-default-export": 0
```

Comment:
Do we need to allow the default export for tests? I don't see any default exports there, and `npm run lint` is locally passing for me. Can we remove this line?